From 348c004dd8110df3ab472a8e8d7c400e1c686ec5 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Sat, 30 Jul 2011 22:27:15 +0800
Subject: [PATCH] Revert "ramconsole/apanic: Ensure ramconsole does not get
 cluttered by apanic threads"

This reverts commit 05ec8148bda5d0206e82eba2357138cc9f02705b.
---
 drivers/misc/apanic.c                 | 4 ----
 drivers/staging/android/ram_console.c | 8 --------
 2 files changed, 12 deletions(-)

diff --git a/drivers/misc/apanic.c b/drivers/misc/apanic.c
index 2f858be7ef59..fde087122473 100644
--- a/drivers/misc/apanic.c
+++ b/drivers/misc/apanic.c
@@ -38,8 +38,6 @@
 #include <linux/workqueue.h>
 #include <linux/preempt.h>
 
-extern void ram_console_enable_console(int);
-
 struct panic_header {
 	u32 magic;
 #define PANIC_MAGIC 0xdeadf00d
@@ -470,8 +468,6 @@ static int apanic(struct notifier_block *this, unsigned long event,
 	if (!threads_offset)
 		threads_offset = ctx->mtd->writesize;
 
-	ram_console_enable_console(0);
-
 	log_buf_clear();
 	show_state_filter(0);
 	threads_len = apanic_write_console(ctx->mtd, threads_offset);
diff --git a/drivers/staging/android/ram_console.c b/drivers/staging/android/ram_console.c
index d735be45f162..8f18a59744cd 100644
--- a/drivers/staging/android/ram_console.c
+++ b/drivers/staging/android/ram_console.c
@@ -146,14 +146,6 @@ static struct console ram_console = {
 	.index	= -1,
 };
 
-void ram_console_enable_console(int enabled)
-{
-	if (enabled)
-		ram_console.flags |= CON_ENABLED;
-	else
-		ram_console.flags &= ~CON_ENABLED;
-}
-
 static void __init
 ram_console_save_old(struct ram_console_buffer *buffer, char *dest)
 {
-- 
2.35.3

